src/app/getting-started/getting-started.component.ts

```diff
@@ -232,4 +261,78 @@ export class GettingStartedComponent implements OnDestroy, OnInit {
       type: type
     } as Notification);
   }
+
+  /**
+   * Lets poll the fabric8-tenant service to see if the kubernetes tenant has connected the
+   * services such as Jenkins into KeyCloak which is typically lazy after the tenant
+   * starts to be created
+   */
+  private kubeTenantConnectPoll() {
```

Comment:
@jstrachan this does not appear to be related to the getting started page or service. I'm going to guess you put it here so it would get run first. If that is the case can you please create a service for this and let's find a better place for it.